setup.py

```diff
@@ -72,7 +72,7 @@
 explanatory text. The Notebook has support for multiple programming
 languages, sharing, and interactive widgets.
 
-Read `the documentation <https://jupyter-notebook.readthedocs.io>`_
+Read `the documentation <https://jupyter-notebook.readthedocs.io/en/stable/>`_
```

Comment:
What if we support other languages in the future? I think readthedocs can potentially redirect to the appropriate labguage version based on locale. I think we would be better off sticking with the original.

Reply:
Alright. Fixing it now!